nntrainer/layers/reduce_mean_layer.cpp

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * @file   reduce_mean_layer.cpp
+ * @date   25 Nov 2021
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug    No known bugs except for NYI items
+ * @brief  This is Reduce Mean Layer Class for Neural Network
+ */
+
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <node_exporter.h>
+#include <reduce_mean_layer.h>
+
```

Comment:
I guess <layer_context.h> should be added here?

Reply:
https://github.com/nnstreamer/nntrainer/pull/1740#discussion_r760095535